pvlib/iotools/sodapro.py

```diff
@@ -0,0 +1,361 @@
+"""Functions to access data from Copernicus Atmosphere Monitoring Service
+    (CAMS) radiation service.
+.. codeauthor:: Adam R. Jensen<adam-r-j@hotmail.com>
+"""
+
+import pandas as pd
+import requests
+import io
+import warnings
+
+
+CAMS_INTEGRATED_COLUMNS = [
+    'TOA', 'Clear sky GHI', 'Clear sky BHI', 'Clear sky DHI', 'Clear sky BNI',
+    'GHI', 'BHI', 'DHI', 'BNI',
+    'GHI no corr', 'BHI no corr', 'DHI no corr', 'BNI no corr']
+
+# Dictionary mapping CAMS Radiation and McClear variables to pvlib names
+CAMS_VARIABLE_MAP = {
+    'TOA': 'ghi_extra',
+    'Clear sky GHI': 'ghi_clear',
+    'Clear sky BHI': 'bhi_clear',
```

Comment:
I'm curious what other folks think about using `bhi` for pvlib.  I assume it is beam horizontal irradiance?  It's a shame that diffuse and direct both start with "d".

Reply:
@kanderso-nrel correct it is beam horizontal irradiance. It's a common issue without any good solution. CAMS calls it beam instead of direct which is one solution. Have you any suggestions for alternatives?

Reply:
I'm +0.1 on "beam" as the term here, and -1 on using "direct" which would require us to get cute with the "horizontal" part of the acronym. The only use of "beam" in pvlib is [this function name ](https://github.com/pvlib/pvlib-python/blob/5b0b15ae2e451ddcd91af5f69f9bfb4595ec57e1/pvlib/irradiance.py#L274).  'bhi' doesn't conflict with other terms, and may help users to translate between CAMS and pvlib terms.

Reply:
I'm fine with bhi_clear. Looks like [pv-terms] does not have a relevant recommendation.